Forms/Schema/FormSchema.php

```diff
@@ -67,10 +67,10 @@ public function getState(Form $form) {
 			$this->getFieldStates($form->Actions())
 		);
 
-		if($form->Message()) {
-			$state['messages'][] = [
```

Comment:
The limitation does keep the API simple though; I will look at allowing multiple messages for the form, but let's keep a single message per field for now.